### ISSUE TITLE
Use ingredients with the highest count first

### ISSUE THREAD

**Issue by rhullah** [CyclopsMC/IntegratedTerminals#135]
#### Issue type:

- :heavy_plus_sign: Feature request

____

#### Short description:

When filling the crafting grid from JEI/EMI/etc it seems when an ingredient matches based on tag (like planks/logs/etc) it chooses the ingredient either randomly or a balance use of the different types. For example, if I'm crafting a vanilla chest and I have 300 oak planks and 5 spruce planks it will often use a mix of both oak and spruce to craft the chest. I want it to use as many oak planks as possible until the the difference between oak and spruce planks either equal or another has a higher count.

I'd prefer this be the default consumption style, but I suppose it could be a configuration setting.

**Comment by rubensworks** [comment]
Thanks for the suggestion!

**Comment by RoarkCats** [comment]
Yes! It's super annoying having to set my autocrafting to use whatever random wood/plank I have tons of instead of the single stack of decorative planks I have for building for ex
